cmd/runhcs/create-scratch.go

```diff
@@ -52,6 +52,11 @@ var createScratchCommand = cli.Command{
 			}
 		} else {
 			opts := uvm.NewDefaultOptionsLCOW("createscratch-uvm", context.GlobalString("owner"))
+
+			// Change the defaults to physical 256MB RAM
+			opts.MemorySizeInMB = 256
```

Comment:
BTW: `256` is also a guess. I ran 100 iterations at `256` with no failures. At `128` I had three guest panics so `256` seems fine.